src/components/Autocomplete/components/ComboBox/tests/ComboBox.test.tsx

```diff
@@ -458,6 +458,34 @@ describe('<ComboBox/>', () => {
       };
     });
 
+    it("will prevent default and select an item on the 'enter' keyup", () => {
+      const spy = jest.fn();
+      const comboBox = mountWithAppProvider(
+        <ComboBox
+          options={options}
+          selected={[]}
+          textField={renderTextField()}
+          onSelect={spy}
+        />,
+      );
+
+      const textField = comboBox.find(TextField);
+
+      async () => {
+        textField.simulate('click');
```

Comment:
We should probably follow web's guidelines here and use `trigger` https://github.com/Shopify/web-foundation/blob/master/Best%20practices/Enzyme.md#event-simulations